src/main/java/org/jboss/tools/intellij/openshift/ui/sandbox/SandboxWorkflowPage.java

```diff
@@ -183,7 +186,7 @@ private void setStatus(String message) {
     }
 
     private void launchJob() {
-        ProgressManager.getInstance().run(new Task.Backgroundable(project, "Login to Red Hat Developer Sandbox", true) {
+        ProgressManager.getInstance().run(new Task.Backgroundable(project, "Login to red hat developer sandbox", true) {
```

Comment:
That's what Jetbrains code analysis suggests. I assume that this is because this isn't a dialog title, this is a status bar message which they apparently dont want to be capitalized. But then, I'd agree to undo "Red Hat" since it's a name.